### ISSUE TITLE
bug: ion-checkbox doesn't read correctly with iOS voiceover

### ISSUE THREAD

**Issue by asselin** [driftyco/ionic#3153]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>ios</span> <span ionic-platform-version>8</span>  <span ionic-webview>webview</span>

<span ionic-description>With VoiceOver turned on in iOS, voiceover does not read an ion-checkbox as a checkbox, and doesn't read it's current state (checked or unchecked).  The root problem seems to be that the directive's template has

````
<label>
   <div>
      <input>
````

The `<div>` between the `<label>` and `<input>` seems to be what's breaking Voiceover.  Removing that `<div>` makes it read correctly.

Tested on iOS 8.1.3</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @asselin!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3153). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3153)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by mhartington** [comment]
Little hesitant to change anything so close to 1.0. Will revisit this post 1.0

**Comment by mlynch** [comment]
Can you try with `ion-checkbox`?

**Comment by asselin** [comment]
@mlynch Sorry, I'm not following your request.  The issue was reported because I was using ion-checkbox.  Are you saying that the HTML that it expands to has been changed to fix this?

**Comment by jgw96** [comment]
@asselin Hey! Is this still an issue with the latest version of Ionic? ionic 1.3.0?

**Comment by jgw96** [comment]
Hello all! As it seems it has been a while since there was any activity on this issue i will be closing it for now. Feel free to comment if you are still running into this issue. Thanks for using Ionic!
